src/test/java/com/microsoft/sqlserver/jdbc/connection/PoolingTest.java

```diff
@@ -139,4 +145,52 @@ public void testConnectionPoolClientConnectionId() throws SQLException {
 
         assertEquals(Id1, Id2, "ClientConnection Ids from pool are not the same.");
     }
+    
+    @Test
+    public void testHikariCP() throws SQLException {
+        HikariConfig config = new HikariConfig();
+        config.setJdbcUrl(connectionString);
+        HikariDataSource ds = new HikariDataSource(config);
+
+        connect(ds);
```

Comment:
@fwi : Thanks for suggestions. We are trying out `Java MicroBenchmark Harness (JMH)` test cases for performance testing. We are figuring out how & when we can open performance test cases. At the same time we want our Travis-CI & Appveyour builds should not be long running.